css/styles.css

```diff
@@ -0,0 +1,115 @@
+body {
+  font-family: 'Ubuntu'; sans-serif
+  text-align: center;
+  background-color: #F2F2F2;
+  color: black;
```

Comment:
@sofiaayala the default text color is black. If its the only text color you are using you could omit this.